Desktop/mainwindow.cpp

```diff
@@ -1470,42 +1470,28 @@ void MainWindow::openGitHubBugReport() const
 		}
 	}
 
-	std::stringstream fillIt;
-
-	try			{ fillIt << "* JASP version: " << AppInfo::version.asString()	<< std::endl; }
-	catch(...)	{ fillIt << "* JASP version: ???\n"; }
-
-	try			{ fillIt <<	"* OS name and version: " << QSysInfo::prettyProductName() << std::endl; }
-	catch(...)	{ fillIt << "* OS name and version: ???\n"; }
-
-	fillIt	<<	"<!--- Please fill in the following fields: -->\n"
-				"* Analysis: \n"
-				"* Bug description:\n"
-				"* Expected behaviour:\n"
-				"<!--- Steps to reproduce means, what actions should we take in JASP to reproduce the bug you encountered? --->\n"
-				"#### Steps to reproduce:\n"
-				"1. Go to '...'\n"
-				"2. Click on '....'\n"
-				"3. Scroll down to '....'\n"
-				"4. See error\n";
-
-	fillIt <<	"\n\n\n"
-				"-----------------------------------------------------------------------\n"
-				"<!--- A note from the developers:\nIf possible please attach your data and/or JASP file to the issue, this makes solving the bug a lot easier."
-				" If you would prefer to not make your data publicly available then you could also mail us.\n"
-				"Note that github requires you to zip the file to upload it here.\n-->\n\n"
-				"### Debug information:\n" << _engineSync->currentStateForDebug();
-
-	try			{ fillIt << "\n[Commit used](" << AboutModel::commitUrl() << ")\n"; }
-	catch(...)	{ fillIt << "Commit couldn't be found\n"; }
+	std::stringstream systemInfo, debugInfo;
+
+	try			{ systemInfo << "* JASP version: " << AppInfo::version.asString()	<< std::endl; }
+	catch(...)	{ systemInfo << "* JASP version: ???" << std::endl; }
+
+	try			{ systemInfo <<	"* OS name and version: " << QSysInfo::prettyProductName() << std::endl; }
+	catch(...)	{ systemInfo << "* OS name and version: ???" << std::endl; }
+
+	try			{ systemInfo << "* Commit used: " << AboutModel::commitUrl() << std::endl; }
+	catch(...)	{ systemInfo << "Commit couldn't be found\n"; }
+
+	try			{ debugInfo << "Debug information: " << _engineSync->currentStateForDebug() << std::endl; }
+	catch(...)	{ debugInfo << "No debug information found"; }
 
 	try
 	{
-		QString percentEncodedIssue = QUrl::toPercentEncoding(tq(fillIt.str()));
+		QString systemInfoStr	= QUrl::toPercentEncoding(tq(systemInfo.str())),
+				debugInfoStr	= QUrl::toPercentEncoding(tq(debugInfo.str()));
 
-		const char * baseIssueUrl = "https://github.com/jasp-stats/jasp-issues/issues/new?labels=bug&title=JASP+crashed&body=";
+		QString baseIssueUrl = "https://github.com/jasp-stats/jasp-issues/issues/new?template=crash-report.yml&title=JASP+crashed";
 
-		QUrl issueUrl = baseIssueUrl + percentEncodedIssue;
+		QUrl issueUrl = baseIssueUrl + "&system-info=" + systemInfoStr + "&log=" + debugInfoStr;
 
 		QDesktopServices::openUrl(issueUrl);
```

Comment:
This openUrl() will open 3 same issues so we should make sure it just open only once.

Reply:
I don't understand this... It opens only 1 issue...

Reply:
I'm not sure where you created a crash, but the fact is that our program crashes randomly. For example if I call `openGitHubBugReport()` inside the function of `MainWindow::setQmlImportPaths()` directly to simulate a crash qml slots , three identical error reports will be opened, which is the same as many user-side crash reports.

Reply:
So you can see: 
![image](https://github.com/user-attachments/assets/3bc2af49-4d90-449b-95c2-2292caa4870b)

Reply:
Oh I see... The setQmlImportPaths was apparently called 3 times: 1 during the main initialization, and 2 times in the initialization of the modules (you use apparently the JASP_TEST_BUILD setting, and you have only 2 modules?) 
But this is an artificial situation. The openGitHubBugReport is called only when an Engine crashed. Maybe the Desktop tried to restore the Engine, and it crashes just afterwards, or all the running Engines crashed simultaneously... In any case, we should get only 1 issue. I'll see what I can do to remove the redundant calls.

Reply:
Yes, your solution works, I have added it, thanks!

Reply:
I dont understand this conversation?

First the three issues was because of the artificial situation. Ok, so its not a problem.
But then Bruno says "remove redundant calls" and adds the code of Shun.

But why? What redundant calls?

Reply:
My example is not appropriate, but in fact there are indeed situations where multiple issues are opened at the same time, see my screenshot above from users.